extensions/issues/MultipleIncorrectSubmissions/MultipleIncorrectSubmissions.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from extensions.issues import base
+
+
+class MultipleIncorrectSubmissions(base.BaseExplorationIssueSpec):
```

Comment:
@pranavsid98 don't we capture what were the incorrect answers that learner tried to enter repeatedly?

Reply:
Yep! I've added that in the documentation for the class.

Reply:
@anmolshkl Oh, right, the actual answers are in the learner actions for the card 'state_name'. This is just an issue overview where we know that the learner answered multiple questions incorrectly at a particular card, and when we need more details (as in, the highlighted learner actions), we consider the answer submitted learner actions at the card specified here.